app/views/posts/show.html.erb

```diff
@@ -35,7 +35,9 @@
           <% else %>
             <div class="sales-box">
               <h2>You can view pro screencasts by subscribing</h2>
+              <% if current_user && current_user.subscribed %>
```

Comment:
This conditional logic doesn't seem right.  We want the button to show if it's an anonymous user.